spec/dor/workflow/response/process_spec.rb

```diff
@@ -62,4 +62,32 @@
 
     it { is_expected.to eq 'default' }
   end
+
+  describe '#context' do
+    subject { instance.context }
+
+    context 'when context exists' do
+      let(:xml) do
+        <<~XML
+          <workflow repository="dor" objectId="druid:mw971zk1113" id="assemblyWF">
+            <process name="start-assembly" laneId="default" context="{&quot;requireOCR&quot;:true}">
+          </workflow>
+        XML
+      end
+
+      it { is_expected.to eq({ 'requireOCR' => true }) }
+    end
+
+    context 'when no context exists' do
+      let(:xml) do
+        <<~XML
+          <workflow repository="dor" objectId="druid:mw971zk1113" id="assemblyWF">
+            <process name="start-assembly" laneId="default" context="">
+          </workflow>
+        XML
+      end
+
+      it { is_expected.to be_nil }
```

Comment:
Is there any value in having context always be a hash, and either it is populated with things or not?

Reply:
Sure, we could do that - no preference either way.

Reply:
adjusted to return an empty hash instead of nil when no context available